plugins/notifiers/slack/templates/AppealApproved.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "type":"section",
+    "text":{
+      "type":"mrkdwn",
+      "text":"Your appeal to {{.resource_name}} with role {{.role}} has been approved"
```

Comment:
let's add formatting to the value either make it bold or use code (backtick)
something like this

Your appeal for `{{.resource_name}}` with role `{{.role}}` has been approved.

Reply:
Done